.travis.yml

```diff
@@ -1,15 +1,20 @@
 sudo: false
 language: node_js
 node_js:
-  - "5.5"
+- '5.5'
```

Comment:
that was the travis cli chaning the file. I'll update this to Node 6 in a new commit.